R/app-server.R

```diff
@@ -378,45 +366,109 @@ app_server <- function(input, output, session) {
       )
     })
 
-    ## Successes box
-    observe({
-      successes <- purrr::map_lgl(res(), function(x) {
-        inherits(x, "check_pass")
-      })
-      output$successes <- renderUI({
-        report_results(res()[successes], emoji_prefix = "check")
-      })
-      reporting_titles$success <- glue::glue("Successes ({sum(successes)})")
-    })
-
-    ## Warnings box
-    observe({
-      warnings <- purrr::map_lgl(res(), function(x) {
-        inherits(x, "check_warn")
-      })
-      output$warnings <- renderUI({
-        report_results(
-          res()[warnings],
-          emoji_prefix = "warning",
-          verbose = TRUE
+    ## Show validation results on clicking "validate"
+    ## Require that the study name is given; give error if not
+    observeEvent(input$"validate_btn", {
+      with_busy_indicator_server("validate_btn", {
+        if (study_name() == "") {
+          stop("Please enter study name.")
+        }
+        ## Require at least one file input
+        validate(
+          need(
+            any(
+              !is.null(indiv()),
+              !is.null(biosp()),
+              !is.null(assay()),
+              !is.null(manifest())
+            ),
+            message = "Please upload some data to view a summary"
```

Comment:
```suggestion
            message = "Please upload some data to validate"
```

Reply:
This message is in the data summary section. I don't believe it's used elsewhere. This is what that section would look like if we changed the message. It makes some sense, but not as much sense as what it is now.

![image](https://user-images.githubusercontent.com/25048342/67409445-708d6380-f56f-11e9-9814-a70a169ae54a.png)

Reply:
The data summary message is elsewhere, this line causes the message to appear if you try to click validate without submitting any data.

![Screen Shot 2019-10-23 at 9 05 31 AM](https://user-images.githubusercontent.com/4452678/67412539-56a24f80-f574-11e9-9eee-6ba5065205a9.png)

Reply:
Oh! Haha, nice. Yeah, that definitely makes sense to change. Thanks for catching that!